providers/tests/system/amazon/aws/example_mwaa.py

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow.models.baseoperator import chain
+from airflow.models.dag import DAG
+from airflow.providers.amazon.aws.operators.mwaa import MwaaTriggerDagRunOperator
+
+from providers.tests.system.amazon.aws.utils import SystemTestContextBuilder
+
+DAG_ID = "example_mwaa"
+
+# Externally fetched variables:
+EXISTING_ENVIRONMENT_NAME_KEY = "ENVIRONMENT_NAME"
+EXISTING_DAG_ID_KEY = "DAG_ID"
+
+
+sys_test_context_task = (
+    SystemTestContextBuilder()
+    # NOTE: Creating a functional MWAA environment is time-consuming and requires
+    # manually creating and configuring an S3 bucket for DAG storage and a VPC with
+    # private subnets which is out of scope for this demo. To simplify this demo and
+    # make it run in a reasonable time, follow these steps in the AWS Console to create
+    # a new MWAA environment with default configuration:
+    # 1. Create an S3 bucket and upload your DAGs to a 'dags' directory
+    # 2. Navigate to the MWAA console
+    # 3. Create an environment, making sure to use the S3 bucket from the previous step
+    # 4. Use the default VPC/network settings (or create a new one using this guide:
+    # https://docs.aws.amazon.com/mwaa/latest/userguide/vpc-create.html)
+    # 5. Select Public network for web server access and click through to creation
```

Comment:
Love this comment!

One nit: A lot of these steps are either included or completely automated by the console wizard, and they may change over time. There is also a quick start doc guide that deploys everything in one CFN template. I would just link to those resources rather than type out the steps here: https://docs.aws.amazon.com/mwaa/latest/userguide/quick-start.html

Reply:
Thanks! I used the comment in `example_ecs.py` as inspiration so I included the steps. For the most part, using the console is straightforward except if the default VPC doesn't have a private subnet, it doesn't make it obvious why it's not letting the user select it. So maybe I'll just shorten this down to include this information